padertorch/contrib/examples/speaker_classification/supervised/data.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+
+from lazy_dataset.database import JsonDatabase
+
+from padertorch.contrib.je.data.transforms import (
+    LabelEncoder, AudioReader, STFT, MelTransform, Collate
```

Comment:
is this STFT something different compared to the STFT in paderbox.transform.STFT?

Reply:
It inherits from paderborn.transform.STFT and modifies the __call__ function to handle dictionary inputs in addition to numpy arrays, i.e., you can directly pass an instance of this class to the `map()` call of lazy_dataset.

Reply:
The question it entails is how much work would it be to write some kind of prepare function which is less dependent on contrib and just uses paderbox and the main padertorch. If this would be a lot of work it is not necessary, I just think it is nice to have less dependency on contrib functions which can change all the time.

Reply:
Updated with commit https://github.com/fgnt/padertorch/pull/57/commits/6988846537bc623d6b26664450e7d7f243454e2f. Only `LabelEncoder` remains dependent of `contrib` since there is no equivalent in the core code.